### PR TITLE
feat(k8s collection): document how to use only docker hub registry

### PR DESCRIPTION
**Pull request by sumo-drosiek**
## Purpose of this pull request

Do not merge - not ready yet. Need to update it and wait for new Helm Chart release.

Document how to use Docker Hub images only for Sumo Logic Kubernetes Collection

It requires new Sumo Logic Kubernetes Collection Release

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/send-data/kubernetes/best-practices.md [hunk 1]
```diff
@@ -1002,3 +1002,106 @@ sumologic:
     ## See https://github.com/SumoLogic/sumologic-otel-collector/blob/main/docs/fips.md for more information.
     addFipsSuffix: false
 ```
+
+## Use only Docker Hub registry
```
Comment (sumo-drosiek) [inline docs/send-data/kubernetes/best-practices.md:1006]:
```suggestion
## Use containers only from Docker Hub registry
```

**Review by kkujawa-sumo — APPROVED** [review]

**Comment by kimsauce** [comment]
@sumo-drosiek @kkujawa-sumo , thanks for submitting this! Checking in as it's been a few weeks. Is this ready to publish or still working on it?

File: docs/send-data/kubernetes/best-practices.md [hunk 2]
```diff
@@ -1002,3 +1002,110 @@ sumologic:
...
+      # Ensure the tag is the same like in Sumo Logic Kubernetes Collection Chart, but without `v` prefix
+      tag: 2.7.0
+```
+
+:::note
+
+Not all of the images are maintained by Sumo Logic and not all the vendors provides their images on Docker Hub.
+We suggest the most similar images but we do not guarantee that they will work exactly the same, but they should.
+
+:::
```
Comment (kimsauce) [inline docs/send-data/kubernetes/best-practices.md:1111]:
```suggestion
:::note
While we recommend the most similar images, it's important to note that not all images are maintained by Sumo Logic, and some vendors may not provide their images on Docker Hub. As a result, while the suggested images should work similarly, we cannot guarantee identical functionality.
:::
```

File: docs/send-data/kubernetes/best-practices.md [hunk 3]
```diff
@@ -1002,3 +1002,110 @@ sumologic:
     ## See https://github.com/SumoLogic/sumologic-otel-collector/blob/main/docs/fips.md for more information.
     addFipsSuffix: false
 ```
+
+## Use containers only from Docker Hub registry
+
+If for some reason you can use only DockerHub as docker registry, we are suggesting to use the following configuration:
```
Comment (kimsauce) [inline docs/send-data/kubernetes/best-practices.md:1008]:
```suggestion
If DockerHub is the only available Docker registry option, we recommend the following configuration:
```

File: docs/send-data/kubernetes/best-practices.md (hunk 2 above)
Comment (sumo-drosiek) [reply docs/send-data/kubernetes/best-practices.md]: I removed this note as we clone all of the images right now

Comment (kimsauce) [reply docs/send-data/kubernetes/best-practices.md]: thank you!

**Review by kimsauce — APPROVED** [review]

**Pull request merged**